src/sentry/static/sentry/app/components/tooltip.tsx

```diff
@@ -233,7 +225,17 @@ class Tooltip extends React.Component<Props, State> {
   }
 }
 
-const TooltipContent = styled('div')<Pick<Props, 'popperStyle'>>`
+// Using an inline-block solves the container being smaller
+// than the elements it is wrapping
+const Container = styled('span')<{
+  containerDisplayMode?: React.CSSProperties['display'];
+}>`
+  ${p => p.containerDisplayMode && `display: ${p.containerDisplayMode}`};
+  max-width: 100%;
+`;
+
+type TooltipContentProps = Pick<Props, 'popperStyle'>;
+const TooltipContent = styled('div')<TooltipContentProps>`
```

Comment:
Why the change here, having this inline seemed fine.

Reply:
yes, it is fine ... I have to investigate because with the current option, this part of the code in my vscode looks strange:
![image](https://user-images.githubusercontent.com/29228205/77514404-14f53800-6e77-11ea-869a-9dc82a59bd95.png)